tests/include/test/helpers.h

```diff
@@ -82,7 +83,7 @@ void mbedtls_test_fail( const char *test, int line_no, const char* filename );
  */
 void mbedtls_test_set_step( unsigned long step );
 
-void mbedtls_test_skip( const char *test, int line_no, const char* filename );
+void mbedtls_test_info_reset( void );
```

Comment:
Please document the function like mbedtls_test_set_step() is documented.

Reply:
Done along with `mbedtls_test_fail` and `mbedtls_test_skip`.